src/typeDefs/inputs.ts

```diff
@@ -2,27 +2,152 @@ import { gql } from "apollo-server-core";
 
 // Place fields alphabetically to ensure easier lookup and navigation.
 export const inputs = gql`
+  input AcceptAdminInput {
+    id: ID!
+  }
+
+  input AcceptMembershipRequestInput {
+    membershipRequestId: ID!
+  }
+
+  input AddLanguageTranslationInput {
+    en_value: String!
+    translation_lang_code: String!
+    translation_value: String!
+  }
+
+  input AddOrganizationImageInput {
+    file: String!
+    organizationId: String!
+  }
+
+  input AddUserImageInput {
+    file: String!
+  }
+
+  input AddUserToGroupChatInput {
+    userId: ID!
+    chatId: ID!
+  }
+
+  input AdminRemoveEventInput {
+    eventId: ID!
+  }
+
+  input AdminRemoveGroupInput {
+    groupId: ID!
+  }
+
+  input blockPluginCreationBySuperadminInput {
+    userId: ID!
+    blockUser: Boolean!
+  }
+
+  input BlockUserInput {
+    organizationId: ID!
+    userId: ID!
+  }
+
+  input CancelMembershipRequestInput {
+    membershipRequestId: ID!
+  }
+
+  input CreateAdminInput {
+    organizationId: ID!
+    userId: ID!
+  }
+
+  input CreateCommentInput {
+    postId: ID!
+    data: CommentInput!
+  }
+
   input CommentInput {
     text: String!
   }
 
-  input createChatInput {
+  input CreateChatInput {
     userIds: [ID!]!
     organizationId: ID!
   }
 
-  input createGroupChatInput {
+  input CreateDonationInput {
+    userId: ID!
+    orgId: ID!
+    payPalId: ID!
+    nameOfUser: String!
+    amount: Float!
+    nameOfOrg: String!
+  }
+
+  input CreateEventInput {
+    title: String!
+    description: String!
+    startDate: Date!
+    endDate: Date
+    startTime: Time
+    endTime: Time
+    allDay: Boolean!
+    recurring: Boolean!
+    recurrance: Recurrance
+    isPublic: Boolean!
+    isRegisterable: Boolean!
+    location: String
+    latitude: Latitude
+    longitude: Longitude
+    organizationId: ID!
+  }
+
+  input CreateGroupInput {
+    title: String
+    description: String
+    organizationId: ID!
+  }
+
+  input CreateGroupChatInput {
     userIds: [ID!]!
     organizationId: ID!
     title: String!
   }
 
+  input CreateMessageChatInput {
+    message: String!
+    receiver: ID!
+  }
+
+  input CreateOrganizationInput {
+    data: OrganizationInput!
+    file: String
```

Comment:
Change these names like `file` to what they actually represents, for example `image`, `video`, `profile_picture` etc., whatever's accurate.